### PR TITLE
Fixed use of "window" instead of "global".

### PR DESCRIPTION
**Pull request by tswindell**
Signed-off-by: Tom Swindell <t.swindell@rubyx.co.uk>

### CODE REVIEW

File: webcrypto-shim.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ module.exports = function webcryptoShim (global) {
         _SubtleCrypto = global.SubtleCrypto || _subtle.constructor || Object,
         _CryptoKey  = global.CryptoKey || global.Key || Object;
 
-    var isEdge = window.navigator.userAgent.indexOf('Edge/') > -1
+    var isEdge = global.navigator.userAgent.indexOf('Edge/') > -1
```
Comment (dignifiedquire) [inline webcrypto-shim.js:22]: I don't think global is defined in all browsers, maybe check for not being undefined in this order

1. global
2. window
3. self

**Review by dignifiedquire — CHANGES_REQUESTED** [review]

File: webcrypto-shim.js (hunk 1 above)
Comment (tswindell) [reply webcrypto-shim.js:22]: global is a local variable, it is the variable that is assigned to window or self.

Comment (dignifiedquire) [reply webcrypto-shim.js:22]: I should read the code I'm reviewing more closely :/

**Comment by dignifiedquire** [comment]
Thank you :octocat:

**Review by dignifiedquire — APPROVED** [review]

**Comment by tswindell** [comment]
Is this closed now then? ;)

**Comment by dignifiedquire** [comment]
latest libp2p-crypto includes this and runs tests in webworkers so should work all good now :)

**Pull request merged**